src/Tribe/Main.php

```diff
@@ -206,6 +208,10 @@ public function bind_implementations() {
 		tribe_singleton( 'tickets.commerce.paypal', new Tribe__Tickets__Commerce__PayPal__Main );
 		tribe_singleton( 'tickets.redirections', 'Tribe__Tickets__Redirections' );
 
+		// Attendee Info
+		tribe_singleton( 'tickets.attendees.info', 'Tribe__Tickets__Attendee_Info', array( 'hook' ) );
```

Comment:
We're trying to move away from just adding things into main and trying to keep things into logically organized service providers.
It would make sense to dedicate a service provider to this (e.g. `Tribe__Tickets__Attendee_Info__Service_Provider`) and in that do all the filtering needed.